src/patient-registration/form-manager.ts

```diff
@@ -0,0 +1,151 @@
+import {
+  PatientIdentifier,
+  FormValues,
+  PatientIdentifierType,
+  AttributeValue,
+  PatientUuidMapType,
+  Patient,
+} from './patient-registration-helper';
+import { generateIdentifier } from './patient-registration.resource';
+import { ConfigObject, interpolateString } from '@openmrs/esm-framework';
+import { Config } from '@testing-library/dom';
+
+export default class FormManager {
+  static createIdentifiers(
+    values: FormValues,
+    patientUuidMap: object,
+    identifierTypes: Array<PatientIdentifierType>,
+    abortController: AbortController,
+    location: string,
+  ) {
+    const identifiers: Array<Promise<PatientIdentifier>> = identifierTypes.map(type => {
```

Comment:
the previous implementation of this mixed promises with objects in it's return value so i decided to just force the return value to always be an array of promises. just to keep things consistent.